<!DOCTYPE html><!--cvcjOqEnh44pDWmtY9X7b--><html lang="en"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link rel="preload" href="/search/_next/static/media/47cbc4e2adbc5db9-s.p.woff2" as="font" crossorigin="" type="font/woff2"/><link rel="stylesheet" href="/search/_next/static/css/e446a64f2ff89daf.css" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" href="/search/_next/static/js/webpack.ccaf2443.js"/><script src="/search/_next/static/js/4bd1b696.100b9d70.js" async=""></script><script src="/search/_next/static/js/1255.90e9842b.js" async=""></script><script src="/search/_next/static/js/main-app.0e7376d5.js" async=""></script><script src="/search/_next/static/js/9692.83f9877c.js" async=""></script><script src="/search/_next/static/js/421.98ca62d9.js" async=""></script><script src="/search/_next/static/js/1759.b13f3ee1.js" async=""></script><script src="/search/_next/static/js/3535.878ceef2.js" async=""></script><script src="/search/_next/static/js/2619.b8db57ac.js" async=""></script><script src="/search/_next/static/js/3820.af314958.js" async=""></script><script src="/search/_next/static/js/574.fd20103e.js" async=""></script><script src="/search/_next/static/js/5738.d28a9943.js" async=""></script><script src="/search/_next/static/js/app/layout.7ef30b9e.js" async=""></script><script src="/search/_next/static/js/690.e023e61b.js" async=""></script><script src="/search/_next/static/js/5332.4ca3e6c6.js" async=""></script><script src="/search/_next/static/js/2410.ec36c5aa.js" async=""></script><script src="/search/_next/static/js/5183.9f1a7545.js" async=""></script><script src="/search/_next/static/js/3055.2a4989ee.js" async=""></script><script src="/search/_next/static/js/8977.d520dad7.js" async=""></script><script src="/search/_next/static/js/app/items/%5Bslug%5D/page.0819d17d.js" async=""></script><meta name="next-size-adjust" content=""/><meta name="emotion-insertion-point" content=""/><link rel="preconnect" href="https://fonts.googleapis.com"/><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous"/><link rel="preconnect" href="https://www.cataloguementalhealth.ac.uk"/><link rel="dns-prefetch" href="https://harmonydata.ac.uk"/><title>Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018</title><meta name="description" content="We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.
The proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. 
The primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. 
Our third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. 
Additional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."/><meta property="og:title" content="Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018"/><meta property="og:description" content="We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.
The proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. 
The primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. 
Our third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. 
Additional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."/><meta property="og:url" content="https://harmonydata.ac.uk/search/items/genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018"/><meta property="og:site_name" content="Academic Resource Discovery"/><meta property="og:locale" content="en_US"/><meta property="og:image" content="https://harmonydata.ac.uk/search/harmony.png"/><meta property="og:image:width" content="1200"/><meta property="og:image:height" content="630"/><meta property="og:image:alt" content="Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018"/><meta property="og:type" content="website"/><meta name="twitter:card" content="summary_large_image"/><meta name="twitter:title" content="Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018"/><meta name="twitter:description" content="We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.
The proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. 
The primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. 
Our third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. 
Additional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."/><meta name="twitter:image" content="https://harmonydata.ac.uk/search/harmony.png"/><link rel="icon" href="/search/favicon.ico" type="image/x-icon" sizes="16x16"/><style>
            /* Ensure immediate rendering with Roboto and fallbacks */
            * { 
              font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !important;
              font-display: swap;
              -webkit-font-smoothing: antialiased;
              -moz-osx-font-smoothing: grayscale;
            }
            body { 
              visibility: visible !important; 
              opacity: 1 !important; 
              margin: 0; 
              padding: 0; 
            }
          </style><script src="/search/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule=""></script><style data-emotion="mui-global v658lt">html{-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;box-sizing:border-box;-webkit-text-size-adjust:100%;}*,*::before,*::after{box-sizing:inherit;}strong,b{font-weight:700;}body{margin:0;color:#1A1A1A;font-size:0.875rem;line-height:1.5;font-family:'Roboto','Roboto Fallback',-apple-system,BlinkMacSystemFont,Segoe UI,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;font-weight:400;background-color:#FFFFFF;}@media (min-width:600px){body{font-size:1rem;}}@media print{body{background-color:#fff;}}body::backdrop{background-color:#FFFFFF;}</style></head><body><div hidden=""><!--$--><!--/$--></div><!--$!--><template data-dgst="BAILOUT_TO_CLIENT_SIDE_RENDERING"></template><div>Loading...</div><!--/$--><script src="/search/_next/static/js/webpack.ccaf2443.js" id="_R_" async=""></script><script>(self.__next_f=self.__next_f||[]).push([0])</script><script>self.__next_f.push([1,"1:\"$Sreact.fragment\"\n2:I[52332,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"default\"]\n3:I[65380,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"AuthProvider\"]\n4:I[41627,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"FirebaseProvider\"]\n5:\"$Sreact.suspense\"\n6:I[92114,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"SearchProvider\"]\n7:I[94049,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"default\"]\n8:I[20190,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/j"])</script><script>self.__next_f.push([1,"s/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"default\"]\n9:I[9766,[],\"\"]\na:I[98924,[],\"\"]\nb:I[74744,[\"9692\",\"static/js/9692.83f9877c.js\",\"421\",\"static/js/421.98ca62d9.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"2619\",\"static/js/2619.b8db57ac.js\",\"3820\",\"static/js/3820.af314958.js\",\"574\",\"static/js/574.fd20103e.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"7177\",\"static/js/app/layout.7ef30b9e.js\"],\"ToastContainer\"]\nd:I[24431,[],\"OutletBoundary\"]\nf:I[15278,[],\"AsyncMetadataOutlet\"]\n11:I[24431,[],\"ViewportBoundary\"]\n13:I[24431,[],\"MetadataBoundary\"]\n15:I[57150,[],\"\"]\n:HL[\"/search/_next/static/media/47cbc4e2adbc5db9-s.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"type\":\"font/woff2\"}]\n:HL[\"/search/_next/static/css/e446a64f2ff89daf.css\",\"style\"]\n"])</script><script>self.__next_f.push([1,"0:{\"P\":null,\"b\":\"cvcjOqEnh44pDWmtY9X7b\",\"p\":\"/search\",\"c\":[\"\",\"items\",\"genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"items\",{\"children\":[[\"slug\",\"genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/search/_next/static/css/e446a64f2ff89daf.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"$undefined\"}]],[\"$\",\"html\",null,{\"lang\":\"en\",\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"meta\",null,{\"name\":\"emotion-insertion-point\",\"content\":\"\"}],[\"$\",\"link\",null,{\"rel\":\"preconnect\",\"href\":\"https://fonts.googleapis.com\"}],[\"$\",\"link\",null,{\"rel\":\"preconnect\",\"href\":\"https://fonts.gstatic.com\",\"crossOrigin\":\"anonymous\"}],[\"$\",\"link\",null,{\"rel\":\"preconnect\",\"href\":\"https://www.cataloguementalhealth.ac.uk\"}],[\"$\",\"link\",null,{\"rel\":\"dns-prefetch\",\"href\":\"https://harmonydata.ac.uk\"}],[\"$\",\"style\",null,{\"dangerouslySetInnerHTML\":{\"__html\":\"\\n            /* Ensure immediate rendering with Roboto and fallbacks */\\n            * { \\n              font-family: \\\"Roboto\\\", -apple-system, BlinkMacSystemFont, \\\"Segoe UI\\\", \\\"Oxygen\\\", \\\"Ubuntu\\\", \\\"Cantarell\\\", \\\"Fira Sans\\\", \\\"Droid Sans\\\", \\\"Helvetica Neue\\\", sans-serif !important;\\n              font-display: swap;\\n              -webkit-font-smoothing: antialiased;\\n              -moz-osx-font-smoothing: grayscale;\\n            }\\n            body { \\n              visibility: visible !important; \\n              opacity: 1 !important; \\n              margin: 0; \\n              padding: 0; \\n            }\\n          \"}}]]}],[\"$\",\"body\",null,{\"children\":[\"$\",\"$L2\",null,{\"children\":[\"$\",\"$L3\",null,{\"children\":[\"$\",\"$L4\",null,{\"children\":[\"$\",\"$5\",null,{\"fallback\":[\"$\",\"div\",null,{\"children\":\"Loading...\"}],\"children\":[\"$\",\"$L6\",null,{\"children\":[[\"$\",\"$L7\",null,{\"sx\":{\"display\":\"flex\",\"flexDirection\":{\"xs\":\"column\",\"md\":\"row\"}},\"children\":[[\"$\",\"$L8\",null,{}],[\"$\",\"$L7\",null,{\"component\":\"main\",\"sx\":{\"flexGrow\":1,\"ml\":{\"xs\":0,\"md\":\"72px\"},\"mt\":{\"xs\":\"64px\",\"md\":0},\"minHeight\":{\"xs\":\"calc(100vh - 64px)\",\"md\":\"100vh\"},\"width\":{\"xs\":\"100%\",\"md\":\"calc(100% - 72px)\"}},\"children\":[\"$\",\"$L9\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$La\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[[\"$\",\"title\",null,{\"children\":\"404: This page could not be found.\"}],[\"$\",\"div\",null,{\"style\":{\"fontFamily\":\"system-ui,\\\"Segoe UI\\\",Roboto,Helvetica,Arial,sans-serif,\\\"Apple Color Emoji\\\",\\\"Segoe UI Emoji\\\"\",\"height\":\"100vh\",\"textAlign\":\"center\",\"display\":\"flex\",\"flexDirection\":\"column\",\"alignItems\":\"center\",\"justifyContent\":\"center\"},\"children\":[\"$\",\"div\",null,{\"children\":[[\"$\",\"style\",null,{\"dangerouslySetInnerHTML\":{\"__html\":\"body{color:#000;background:#fff;margin:0}.next-error-h1{border-right:1px solid rgba(0,0,0,.3)}@media (prefers-color-scheme:dark){body{color:#fff;background:#000}.next-error-h1{border-right:1px solid rgba(255,255,255,.3)}}\"}}],[\"$\",\"h1\",null,{\"className\":\"next-error-h1\",\"style\":{\"display\":\"inline-block\",\"margin\":\"0 20px 0 0\",\"padding\":\"0 23px 0 0\",\"fontSize\":24,\"fontWeight\":500,\"verticalAlign\":\"top\",\"lineHeight\":\"49px\"},\"children\":404}],[\"$\",\"div\",null,{\"style\":{\"display\":\"inline-block\"},\"children\":[\"$\",\"h2\",null,{\"style\":{\"fontSize\":14,\"fontWeight\":400,\"lineHeight\":\"49px\",\"margin\":0},\"children\":\"This page could not be found.\"}]}]]}]}]],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]}]]}],[\"$\",\"$Lb\",null,{\"position\":\"bottom-right\"}]]}]}]}]}]}]}]]}]]}],{\"children\":[\"items\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L9\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$La\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L9\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$La\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$Lc\",null,[\"$\",\"$Ld\",null,{\"children\":[\"$Le\",[\"$\",\"$Lf\",null,{\"promise\":\"$@10\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[[\"$\",\"$L11\",null,{\"children\":\"$L12\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]],[\"$\",\"$L13\",null,{\"children\":[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$5\",null,{\"fallback\":null,\"children\":\"$L14\"}]}]}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$15\",[]],\"s\":false,\"S\":true}\n"])</script><script>self.__next_f.push([1,"16:I[41402,[\"9692\",\"static/js/9692.83f9877c.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"690\",\"static/js/690.e023e61b.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"5332\",\"static/js/5332.4ca3e6c6.js\",\"2410\",\"static/js/2410.ec36c5aa.js\",\"5183\",\"static/js/5183.9f1a7545.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"3055\",\"static/js/3055.2a4989ee.js\",\"8977\",\"static/js/8977.d520dad7.js\",\"6387\",\"static/js/app/items/%5Bslug%5D/page.0819d17d.js\"],\"\"]\n17:T1057,"])</script><script>self.__next_f.push([1,"{\"@context\":\"https://schema.org/\",\"@type\":\"Dataset\",\"name\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\",\"description\":\"We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.\\nThe proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. \\nThe primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. \\nOur third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. \\nAdditional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts.\",\"url\":\"https://harmonydata.ac.uk/search/items/genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\",\"identifier\":[\"http://dx.doi.org/10.5255/UKDA-SN-853526\"],\"keywords\":[\"GENETICS\",\"BLOOD SAMPLES\",\"TWINS\",\"EPIGENETICS\",\"EARLY CHILDHOOD\"],\"temporalCoverage\":\"2015-10-01/2018-09-30\"}"])</script><script>self.__next_f.push([1,"c:[\"$\",\"$5\",null,{\"fallback\":[\"$\",\"div\",null,{\"children\":\"Loading...\"}],\"children\":[[\"$\",\"$L16\",null,{\"strategy\":\"beforeInteractive\",\"id\":\"structured-data\",\"type\":\"application/ld+json\",\"dangerouslySetInnerHTML\":{\"__html\":\"$17\"}}],\"$L18\"]}]\n"])</script><script>self.__next_f.push([1,"19:I[78977,[\"9692\",\"static/js/9692.83f9877c.js\",\"1759\",\"static/js/1759.b13f3ee1.js\",\"690\",\"static/js/690.e023e61b.js\",\"3535\",\"static/js/3535.878ceef2.js\",\"5332\",\"static/js/5332.4ca3e6c6.js\",\"2410\",\"static/js/2410.ec36c5aa.js\",\"5183\",\"static/js/5183.9f1a7545.js\",\"5738\",\"static/js/5738.d28a9943.js\",\"3055\",\"static/js/3055.2a4989ee.js\",\"8977\",\"static/js/8977.d520dad7.js\",\"6387\",\"static/js/app/items/%5Bslug%5D/page.0819d17d.js\"],\"default\"]\n1a:Te73,"])</script><script>self.__next_f.push([1,"We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.\nThe proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. \nThe primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. \nOur third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. \nAdditional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."])</script><script>self.__next_f.push([1,"18:[\"$\",\"$L19\",null,{\"study\":{\"dataset_schema\":{\"@context\":\"https://schema.org/\",\"@type\":\"Dataset\",\"name\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\",\"description\":\"$1a\",\"url\":[\"https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=853526\",\"https://reshare.ukdataservice.ac.uk/853526\"],\"keywords\":[\"GENETICS\",\"BLOOD SAMPLES\",\"TWINS\",\"EPIGENETICS\",\"EARLY CHILDHOOD\"],\"identifier\":[\"http://dx.doi.org/10.5255/UKDA-SN-853526\"],\"includedInDataCatalog\":[{\"@type\":\"DataCatalog\",\"name\":\"UK Data Service\",\"url\":\"https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=853526\"}],\"sponsor\":[{\"@type\":\"Organization\",\"name\":\"Economic and Social Research Council\"}],\"temporalCoverage\":\"2015-10-01/2018-09-30\"},\"extra_data\":{\"country_codes\":[\"GB\"],\"harmony_id\":\"ukds/853526\",\"start_year\":2015,\"end_year\":2018,\"data_access\":\"The Data Collection is available for download to users registered with the UK Data Service.\",\"urls\":[\"https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=853526\",\"https://reshare.ukdataservice.ac.uk/853526\"],\"geographic_coverage\":\"\",\"name\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\",\"slug\":\"genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\",\"num_variables\":null,\"dois\":[\"http://dx.doi.org/10.5255/UKDA-SN-853526\"],\"sex\":\"all\",\"instruments\":[],\"study_design\":[\"longitudinal\",\"cohort\"],\"ai_summary\":null,\"duration_years\":3,\"resource_type\":\"dataset\",\"genetic_data_collected\":true,\"source\":[\"ukds\"],\"language_codes\":[\"en\"],\"uuid\":\"f1b86f2047e3c0247db3e4d221c8a171\"},\"distance\":0,\"score\":0,\"parent\":{},\"ancestors\":[]}}]\n"])</script><script>self.__next_f.push([1,"12:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1\"}]]\ne:null\n"])</script><script>self.__next_f.push([1,"1b:Te73,"])</script><script>self.__next_f.push([1,"We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.\nThe proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. \nThe primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. \nOur third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. \nAdditional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."])</script><script>self.__next_f.push([1,"10:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\"}],[\"$\",\"meta\",\"1\",{\"name\":\"description\",\"content\":\"$1b\"}],\"$L1c\",\"$L1d\",\"$L1e\",\"$L1f\",\"$L20\",\"$L21\",\"$L22\",\"$L23\",\"$L24\",\"$L25\",\"$L26\",\"$L27\",\"$L28\",\"$L29\",\"$L2a\",\"$L2b\"],\"error\":null,\"digest\":\"$undefined\"}\n"])</script><script>self.__next_f.push([1,"2e:I[80622,[],\"IconMark\"]\n1c:[\"$\",\"meta\",\"2\",{\"property\":\"og:title\",\"content\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\"}]\n2c:Te73,"])</script><script>self.__next_f.push([1,"We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.\nThe proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. \nThe primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. \nOur third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. \nAdditional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."])</script><script>self.__next_f.push([1,"1d:[\"$\",\"meta\",\"3\",{\"property\":\"og:description\",\"content\":\"$2c\"}]\n1e:[\"$\",\"meta\",\"4\",{\"property\":\"og:url\",\"content\":\"https://harmonydata.ac.uk/search/items/genome-wide-dna-methylation-profiles-across-four-uk-population-based-cohorts-2015-2018\"}]\n1f:[\"$\",\"meta\",\"5\",{\"property\":\"og:site_name\",\"content\":\"Academic Resource Discovery\"}]\n20:[\"$\",\"meta\",\"6\",{\"property\":\"og:locale\",\"content\":\"en_US\"}]\n21:[\"$\",\"meta\",\"7\",{\"property\":\"og:image\",\"content\":\"https://harmonydata.ac.uk/search/harmony.png\"}]\n22:[\"$\",\"meta\",\"8\",{\"property\":\"og:image:width\",\"content\":\"1200\"}]\n23:[\"$\",\"meta\",\"9\",{\"property\":\"og:image:height\",\"content\":\"630\"}]\n24:[\"$\",\"meta\",\"10\",{\"property\":\"og:image:alt\",\"content\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\"}]\n25:[\"$\",\"meta\",\"11\",{\"property\":\"og:type\",\"content\":\"website\"}]\n26:[\"$\",\"meta\",\"12\",{\"name\":\"twitter:card\",\"content\":\"summary_large_image\"}]\n27:[\"$\",\"meta\",\"13\",{\"name\":\"twitter:title\",\"content\":\"Genome-wide DNA methylation profiles across four UK population based cohorts 2015-2018\"}]\n2d:Te73,"])</script><script>self.__next_f.push([1,"We obtained newly-generated genome-wide DNA methylation profiles from whole blood in 240 individuals from the TwinsUK cohort. DNA methylation levels were profiled at 38-86 years of age using the Infinium MethylationEPIC BeadChip. Epigenetic mechanisms such as DNA methylation are key regulators of gene function. Epigenetic signals are malleable and can change in response to internal and external stimuli. The epigenome thereby provides a mechanism of interaction between the genome with the environment, and we hypothesize that early life stimuli and exposures over the life course leave an epigenetic mark.\nThe proposal will explore DNA methylation in 4,024 samples from four British cohort studies (the MRC National Survey for Health and Development (NSHD) or 1946 birth cohort, the National Child Development Study (NCDS) or 1958 birth cohort, the 1970 British Cohort Study (BCS70), and the TwinsUK cohort (TwinsUK)) in order to identify epigenetic signatures of early life experience and exposure to social, environmental, and biological stimuli over the life course, linking findings to changes in physical and cognitive function during ageing. Each study captures a range of early life experiences, longitudinal health measures and lifestyle questionnaire data from adult life, DNA samples collected at single or multiple time-points, and in a sample subset, multiple genomic data for follow-up analyses. \nThe primary research design is a prospective analysis of longitudinal data across 2,336 blood and 1,688 buccal samples from the four cohorts. The first aim of the research will be to establish whether biological, environmental, and social stimuli in early life and over the life course result in detectable differences in DNA methylation profiles in adults. We will consider whether there are epigenetic associations with a number of factors including biological, environmental and social factors in utero and in early childhood (e.g. birthweight, childhood growth, maternal smoking during pregnancy, nutrition, parental socioeconomic position), and in later life: (e.g. smoking, alcohol consumption, physical activity, diet, stressful events, adult socioeconomic position). The second aim is to assess whether there are differences between cohorts in DNA methylation patterns, comparing samples containing individuals born in different years (1946, 1958 and 1970), accounting for age, since these may reflect differences between cohorts in environmental and social influences in early life. \nOur third aim is to explore the role of epigenetics in healthy functional ageing by applying a two-fold approach. First, we will compare epigenetic signatures to longitudinal functional health trajectories throughout life across cohorts, across cell types (blood and buccal), and across different age categories. We will explore whether DNA methylation signatures of early life experiences can mediate functional ageing trajectories (such as cardiovascular, lung and cognitive function), and whether they can be reversed in response to social and environmental exposures in later life. Our fourth aim is to apply a new approach to estimate biological age, the epigenetic clock, to assess the rate of epigenetic ageing and relate it to early life stimuli and longitudinal biomedical, social, and environmental trajectories. \nAdditional analyses in subsamples will include DNA methylation profiling at multiple time-points to estimate reversibility of DNA methylation, cross-tissue comparison across blood, buccal, and additional tissues, and gene expression analysis for functional interpretation. Replication will be pursued in 3,970 samples from four independent UK-population-based cohorts."])</script><script>self.__next_f.push([1,"28:[\"$\",\"meta\",\"14\",{\"name\":\"twitter:description\",\"content\":\"$2d\"}]\n29:[\"$\",\"meta\",\"15\",{\"name\":\"twitter:image\",\"content\":\"https://harmonydata.ac.uk/search/harmony.png\"}]\n2a:[\"$\",\"link\",\"16\",{\"rel\":\"icon\",\"href\":\"/search/favicon.ico\",\"type\":\"image/x-icon\",\"sizes\":\"16x16\"}]\n2b:[\"$\",\"$L2e\",\"17\",{}]\n14:\"$10:metadata\"\n"])</script></body></html>